daemon/api_model.go

```diff
@@ -21,25 +21,45 @@ package daemon
 
 import (
 	"encoding/json"
+	"errors"
 	"net/http"
+	"net/url"
 
 	"github.com/snapcore/snapd/asserts"
 	"github.com/snapcore/snapd/overlord/auth"
 	"github.com/snapcore/snapd/overlord/devicestate"
 )
 
-var modelCmd = &Command{
-	Path: "/v2/model",
-	POST: postModel,
-	// TODO: provide GET here too once we decided on the details of the API
-}
+var (
+	serialModelCmd = &Command{
+		Path: "/v2/model/serial",
+		GET:  getSerial,
```

Comment:
this should have a `UserOK: true`, as it's OK for a user to GET this endpoint.

Reply:
Done